### PR TITLE
Russian Localization

### PR DESCRIPTION
**Pull request by aleksandrshoshiashvili**

### CODE REVIEW

**Comment by nickoneill** [comment]
Thanks @aleksandrshoshiashvili !

File: files/Russian.string [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+"LocationInUse" = "Геолокация во время использования приложения";
+"Notifications" = "Уведомления";
+"Microphone" = "Микрофон";
+"Camera" = "Камера";
+"Photos" = "Фото";
+"Reminders" = "Напоминания";
+"Bluetooth" = "Bluetooth";
+"Motion" = "Motion";
+//
+"Allowed Contacts" = "Контакты рарешены";
```
Comment (max-potapov) [inline files/Russian.string]: "раЗрешены" – typo in entire section.

File: files/Russian.string [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+"Allowed Events" = "События рарешены";
+"Allowed Location" = "Геолокация рарешена";
+"Allowed Notifications" = "Уведомления рарешены";
+"Allowed Microphone" = "Микрофон рарешен";
+"Allowed Camera" = "Камера рарешена";
+"Allowed Photos" = "Фото рарешены";
+"Allowed Reminders" = "Напоминания рарешены";
+"Allowed Bluetooth" = "Bluetooth рарешены";
```
Comment (max-potapov) [inline files/Russian.string]: "разрешен"

File: files/Russian.string [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+"Allowed Motion" = "Motion рарешены";
```
Comment (max-potapov) [inline files/Russian.string]: "разрешен"

File: files/Russian.string [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+"Denied Microphone" = "Доступ к Микрофону отклонен";
+"Denied Camera" = "Доступ к Камере отклонен";
+"Denied Photos" = "Доступ к Фото отклонен";
+"Denied Reminders" = "Доступ к Напоминаниям отклонен";
+"Denied Bluetooth" = "Доступ к Bluetooth отклонен";
+"Denied Motion" = "Доступ к Motion отклонен";
+//
+"Contacts Disabled" = "Контакты отключены";
+"Events Disabled" = "События отключены";
+"Location Disabled" = "Геолокация отключены";
```
Comment (max-potapov) [inline files/Russian.string]: "отключена"

**Review by max-potapov — COMMENTED** [review]
typos

**Comment by nickoneill** [comment]
Thanks @max-potapov and @aleksandrshoshiashvili!

**Pull request merged**